graspy/simulations/simulations_corr.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from graspy.simulations import sample_edges
+
+
+def sample_edges_corr(P, Rho, directed, loops):
+    """
+    Generate a pair of correlated graphs with Bernoulli distribution.
+    Both G1 and G2 are binary matrices. 
+
+    Parameters
+    ----------
+    P: np.ndarray, shape (n_vertices, n_vertices)
+        Matrix of probabilities (between 0 and 1) for a random graph
+    Rho: np.ndarray, shape (n_vertices, n_vertices)
```

Comment:
Can you call this `R` or something like that? Typically in python variables should be [snake case](https://en.wikipedia.org/wiki/Snake_case) but often this is neglected in the case of matrices (which are sometimes just capital letters) but otherwise I wouldn't have a variable start with capital letter unless it is a class